storage-access.bs

```diff
@@ -147,6 +147,12 @@ partial interface Document {
 };
 </pre>
 
+Let |get unpartitioned cookie enabled state| be an algorithm that, given {{Document}} |doc| and [=cookie store=], runs the following steps:
+1. Let |global| be |doc|'s [=relevant global object=].
+1. Let |cookie enabled state| be the result of [=queuing a global task=] on the [=cookie store=] with |global|'s [=environment/has storage access=].
```

Comment:
You can't queue a task on the cookie store, only on a task source, see https://html.spec.whatwg.org/multipage/webappapis.html#task-source for some info (but yes it's complicated, the important bit here is that you need a task source to queue tasks, which is what you do when you want to manipulate renderer state after stepping into the browser process using "in parallel").

However, it's not entirely clear to me what this step does and why it's needed. I suppose this is still WIP and you would list out steps to get unpartitioned cookie state here?

Reply:
PTAL the latest version [Add more steps into "get unpartitioned cookies enabled state"](https://github.com/shuranhuang/storage-access/pull/1/commits/87cd22d84fac757feb0a6d8b5699a892634c30ac), in which I did the following:
1. made the `get unpartitioned cookie enabled state` algorithm synchronous and incorporate more checks;
2. in hSA, added back the step that queues a task to `permissions task source` to hopefully minimize the changes;
3. in rSA, moved the step that calls `get unpartitioned cookie enabled state` up so that it does not run `in parallel`.